web/src/app/home/PayWithPayPalComponent.js

```diff
@@ -55,10 +54,13 @@ class PayWithPayPalComponent extends Component {
           // Make a call to the REST api to execute the payment. Unfortunatly 'paypal-checkout' service has a bug
           // that prevents executing payments when payee isnt the client's paypal account.
           // So we must do that on our backend. See this bug: https://github.com/paypal/paypal-checkout/issues/464
-          return axios.get(process.env.REACT_APP_API_HOST + 'donations/execute/' + paymentId).then((response) => {
-            console.log(response)
+          return RestClient.GET('donations/execute/' + paymentId).then((response) => {
+            console.log(response);
             window.alert('Payment Complete!');
-          });
+          }, (error) => {
+            console.log(error);
```

Comment:
I think its best to move 'console.log(error);' into the error case for all functions inside RestClient.js

I think we'd want that logging for all calls that fail